hack/get-kustomize.sh

```diff
@@ -5,21 +5,24 @@
 
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 KUSTOMIZE_VERSION="v3.9.1"
+_BINDIR=$THIS_DIR/.bin
 
 # Check if you have kustomize installed, or download it.
 # please note that different versions of kustomize can give different results
 check_or_install_kustomize() {
     # Check if there is already a kustomize binary in $_BINDIR and if yes, check
     # if the version matches the expected one.
-    local kustomize="$(PATH=$THIS_DIR command -v kustomize)"
+    local kustomize="$(PATH=$_BINDIR command -v kustomize)"
     if [ -x "$kustomize" ]; then
         >&2 echo "Found "$kustomize" version "`$kustomize version --short`
         echo $kustomize
         return 0
     fi
     local kustomize_url="https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz"
     curl -sLo kustomize.tar.gz "${kustomize_url}" || return 1
```

Comment:
`curl -sLo kustomize.tar.gz "${kustomize_url}" || return 1`
should this be
`curl -sLo $_BINDIR/kustomize.tar.gz "${kustomize_url}" || return 1`
and moved to line 24?

otherwise I think line 24 will fail. is my understanding right?

Reply:
Line 21 downloads in the local directory, and line 24 expands it in $_BINDIR
But since I checked it locally but it seems I did not check it in my github profile I will verify.

Reply:
It appears to be working fine there too: https://github.com/salv-orlando/nsx-container-plugin-operator/runs/1938639109?check_suite_focus=true

Reply:
my bad. I though tar changes the directory before reading the tarball also https://www.gnu.org/software/tar/manual/html_node/Option-Summary.html#SEC42